db/migrations/20190726141224_create_flap_table_and_triggers.sql

```diff
@@ -134,32 +174,31 @@ VALUES (new_diff.bid_id,
         flap_bid_bid_before_block(new_diff.bid_id, new_diff.address_id, new_diff.header_id),
         (SELECT api.epoch_to_datetime(block_timestamp) FROM diff_block),
         flap_bid_time_created(new_diff.address_id, new_diff.bid_id))
-ON CONFLICT (block_number, bid_id, address_id) DO UPDATE SET guy = new_diff.guy;
+ON CONFLICT (block_number, bid_id, address_id) DO UPDATE SET guy = new_diff.guy
 $$
     LANGUAGE sql;
--- +goose StatementEnd
```

Comment:
Did you intend to remove this? Curious because I don't see a `StatementBegin` removed above, but I see that there is one immediately below

Reply:
Yeah this was a `StatementEnd` without a matching `StatementBegin`; it was actually causing a weird situation where tests were failing in CI but for some reason I couldn't get the same failure locally. Took a minute to troubleshoot what was going on.

As far as I can tell, Goose requires the `StatementBegin`s and `-End`s for `plpgsql` queries, and not for vanilla `sql`. Although it doesn't complain if you include them when they aren't necessary aside from the aforementioned situation where you have a `Begin` w/o an `End` or vice-versa.

Reply:
Ah that makes sense - nice catch! Maybe worth creating a story to see if we can isolate the consequences of them missing/verify they're there?

Reply:
I might be misunderstanding you, but in my experience the migrations totally fail if you don't have those statements where they're necessary. So it _shouldn't_ be necessary to manually go through and verify that they're there, although I can see a case for making sure they're _not_ there where they're unnecessary, for the sake of reducing clutter.

Reply:
Ok cool - as long as things break when broken 👍